api/util/apps.go

```diff
@@ -11,6 +11,8 @@ import (
 
 const (
 	ApplicationNameLabel = "application.apps.nine.ch/name"
+	ManagedByAnnotation  = "app.kubernetes.io/managed-by"
+	NctlName             = "nctl"
```

Comment:
We should probably have this const somewhere we can use it from all packages? Grepping the project I already found two existing instances of the command name `"nctl"`:

```
main.go
54:             kong.Name("nctl"),

auth/login.go
28:     NctlExtensionName = "nctl"
```

Not yet sure where it fits best though.

Reply:
I put it into the `util` package, but I am open for any other location